### DISCUSSION TITLE
How to minify the css code from 'styled-components'?

### DISCUSSION THREAD

**Discussion by podeig** [parcel-bundler/parcel#9088] [Q&A] [answered]
Hi, Is it possible to minify the css code from 'styled-components'? 
How? Thank you! :-)

**THE SOURCE CODE**

![image](https://github.com/parcel-bundler/parcel/assets/32296171/b149875d-d0b9-4890-98fc-1e579e498a06)

**THE BUILT CODE**

![image](https://github.com/parcel-bundler/parcel/assets/32296171/7cabd00f-4f9e-4697-8110-1858d565b4a7)

**Comment by mischnic** [comment, marked as answer]
Currently not, but it would be possible with the right plugin. See https://github.com/parcel-bundler/parcel/discussions/8504

**Reply by podeig** [reply]
I tried the plugins you related to, but it did not work in my case, unfortunately. Thank you for the answer!
